### PR TITLE
testing/gdal: upgrade to 3.0.0

### PR DESCRIPTION
**Pull request by abellgithub**
https://gdal.org/
Update GDAL to version 3 to match proj version 6.

### CODE REVIEW

File: testing/gdal/APKBUILD [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=gdal
-pkgver=2.4.0
+pkgver=3.0.0
 pkgrel=1
 pkgdesc="A translator library for raster and vector geospatial data formats"
 url="http://gdal.org"
-arch="x86 x86_64"
+arch="x86_64"
 license="MIT"
 depends=""
```
Comment (maxice8) [inline testing/gdal/APKBUILD]: empty variables can be removed for brevity

File: testing/gdal/APKBUILD [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Contributor: Trevor R.H. Clarke <trevor@notcows.com>
 # Maintainer: Trevor R.H. Clarke <trevor@notcows.com>
 pkgname=gdal
-pkgver=2.4.0
+pkgver=3.0.0
 pkgrel=1
 pkgdesc="A translator library for raster and vector geospatial data formats"
 url="http://gdal.org"
```
Comment (maxice8) [inline testing/gdal/APKBUILD]: this url is now https://

**Comment by maxice8** [comment]
hexer
mapnik
mapserver
openscenegraph
pdal
postgis

these packages need to be rebuilt due to ABI changes

**Comment by maxice8** [comment]
Please add commits bumping the pkgrel variables of the packages listed above, it is required that they be rebuilt.

**Comment by abellgithub** [comment]
Failure seems due to rebuild of the dependent packages exceeding time.

**Comment by abellgithub** [comment]
Please let me know if there's something else I need to do before this can be merged.  Thx.

**Comment by maxice8** [comment]
There are conflicts with the master branch

**Pull request closed without merging**